packages/gestalt/src/Popover.js

```diff
@@ -1,24 +1,60 @@
 // @flow strict
 import type { Node } from 'react';
 import Controller from './Controller.js';
-import { type Role } from './Contents.js';
 
 type Color = 'blue' | 'orange' | 'red' | 'white' | 'darkGray';
 type Size = 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'flexible' | number;
 type IdealDirection = 'up' | 'right' | 'down' | 'left';
+type Role = 'dialog' | 'listbox' | 'menu';
 
 type Props = {|
+  /**
+   * The reference element, typically [Button](https://gestalt.pinterest.systems/button) or [IconButton](https://gestalt.pinterest.systems/iconbutton), that Popover uses to set its position
+   */
   anchor: ?HTMLElement,
+  /**
+   * The content shown in Popover
+   */
   children?: Node,
+  /**
+   * The background color of Popover. See the [color and caret](https://gestalt.pinterest.systems/popover#Color-and-caret) variant to learn more.
+   */
   color?: Color,
+  /**
+   *
+   */
   handleKeyDown?: (event: SyntheticKeyboardEvent<HTMLElement>) => void,
+  /**
+   * Unique id to identify each Popover. Used for [accessibility](https://gestalt.pinterest.systems/popover#ARIA-attributes) purposes
+   */
   id?: string,
+  /**
+   * Specifies the preferred position of Popover relative to its anchor element. See the [ideal direction](https://gestalt.pinterest.systems/popover#Ideal-direction) variant to learn more.
+   */
   idealDirection?: IdealDirection,
+  /**
+   * Callback fired when Popover requests to be closed. Must be used to control Popover’s on/off display state.
+   */
   onDismiss: () => void,
+  /**
+   * Properly positions Popover relative to its anchor element. Set to false when used within [Layer](https://gestalt.pinterest.systems/layer). See the [with Layer](https://gestalt.pinterest.systems/popover#With-layer) variant to learn more.
+   */
   positionRelativeToAnchor?: boolean,
+  /**
+   * The underlying ARIA role for Popover. See the [accessibility](https://gestalt.pinterest.systems/popover#ARIA-attributes) section for more info.
+   */
   role?: Role,
+  /**
+   * Puts the focus on Popover when it’s triggered. See [accessibility](https://gestalt.pinterest.systems/popover#Accessibility) to learn more.
+   */
   shouldFocus?: boolean,
+  /**
+   * Shows a caret on Popover. See the [color and caret](https://gestalt.pinterest.systems/popover#Color-and-caret) variant to learn more.
+   */
   showCaret?: boolean,
```

Comment:
Some of the default values aren't being shown in the table, as with this prop. You need to move the `props` destructuring into the function signature for docgen to pick up on it.

Reply:
DONE